rsconf/component/__init__.py

```diff
@@ -80,6 +84,55 @@ def build_write(self):
         self._root_bash.extend(self._root_bash_aux)
         self.buildt.write_root_bash(self.name, self._root_bash)
 
+    def gen_host_and_identity_ssh_keys(
+        self, j2_ctx, visibility, password_filename=None
```

Comment:
I think just "encrypt_identity=False" would be better, and have the password_filename be `f"{self.module_name}_ssh_passphrase.json"`. It's not a password, and it isn't "devbox_auth". It's a passphrase for the ssh identity key, but that's long so ssh_passphrase is clear, I think.

Reply:
5cf71b2